### DISCUSSION TITLE
Use with react-table and filters causes everything to be marked dirty

### DISCUSSION THREAD

**Discussion by dannyshaw** [react-hook-form/react-hook-form#10235] [Q&A]
I'm using react-table to render a form. There a 3 columns that have editable fields in them. 
They're rendered using Controller

When a filter is applied to the table, the instant it is applied, all fields on all rows (including visible rows) are marked as dirty.
I've tried `shouldUnregister` but made no difference.

Any ideas how to avoid this?

**Comment by Moshyfawn** [comment]
Completely depends on what you're doing in your code. Do you mind creating a minimal reproduction sandbox?
